langlib/jballerina.java/src/main/ballerina/stackFrameImpl.bal

```diff
@@ -42,3 +42,29 @@ public readonly class StackFrameImpl {
         self.lineNumber = lineNumber;
     }
 }
+
+# Implementation for the `runtime.StackFrame`.
+#
+# + callableName - Callable name
+# + fileName - File name
+# + lineNumber - Line number
+public readonly class StackFrameImplForSingleBalFile {
+
+    public string callableName;
```

Comment:
In StackFrameImpl we can add *StackFrameImplForSingleBalFile and remove callableName, linename and filename

Reply:
We decided to move with single class approach with `string?` field type for `moduleName`